app/AppKernel.php

```diff
@@ -27,10 +27,13 @@ public function registerBundles()
             // Sonata
             new Sonata\CoreBundle\SonataCoreBundle(),
             new Sonata\BlockBundle\SonataBlockBundle(),
-            new Knp\Bundle\MenuBundle\KnpMenuBundle(),
             new Sonata\DoctrineORMAdminBundle\SonataDoctrineORMAdminBundle(),
             new Sonata\AdminBundle\SonataAdminBundle(),
 
+            // KnpLabs
+            new Knp\Bundle\MenuBundle\KnpMenuBundle(),
```

Comment:
I don't know. It was already there. I just moved it from the Sonata section to Knp section.